project/src/main/webapp/index.html

```diff
@@ -12,7 +12,7 @@ <h1>Project Deets</h1>
       <p>We're going to write a lot of code. Click <button onclick="addMessage()"><b>here</b></button> for advice</p>
       <div id="greeting-container"></div>
 
-    <p><a href="addEvent.html">Click this link!</a></p>
+    <p><a href="start-page.html">Click this link to start!</a></p>
```

Comment:
I think we can delete the current index.html and rename `start-page.html` to be index.html

Reply:
That way it directly arrives at start page when we load URL

Reply:
Will be handled in @andy-keene 's PR